.github/workflows/npm-audit.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: npm audit
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'npm-audit'
+
+jobs:
+  audit-root:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+
+    - name: npm audit (root)
+      run: npm audit
+
+  audit-lib:
```

Comment:
Is this job needed? Won't lerna run the script against the repos in the first one?

Reply:
No, `npm audit` just runs against the root. I added `npm run audit` as a way to easily to this all locally, but doing them individually here to make it clear which ones fail (given when `npm run audit` is run, as soon as one fails, it stops running all of them).

Reply:
Oh gotcha - missed the nuance between `npm audit` and `npm run audit`